.gitignore

```diff
@@ -36,18 +36,6 @@ local.properties
 *.iml
 *.hprof
 
-# Nx/Builds
-.docusaurus
```

Comment:
agree that we probably dont need all of these now but i dont think we want to remove all of them. eg building docs still produces a `.docusaurus` directory. we should probably revisit to see what exactly can be removed & just add the package lock for now

Reply:
Yeah, very good, added those back.